todoman/configuration.py

```diff
@@ -31,6 +47,8 @@ def load_config():
 
 
 def _load_config_impl(path):
-    config = ConfigParser(interpolation=None)
-    config.read(path)
-    return config
+    with open(path) as conffile:
+        config = toml.loads(conffile.read())
+    # TODO: Validate required fields here (ie: path)
```

Comment:
If you're adding dependencies already you might as well use JSON-Schema for this